pkg/controllers/cloud/networkpolicy_sync.go

```diff
@@ -261,6 +261,7 @@ func (r *NetworkPolicyReconciler) syncWithCloud() {
 	if r.bookmarkCnt < npSyncReadyBookMarkCnt {
 		return
 	}
+	log.V(1).Info("Started synchronizing security groups in controller with cloud")
```

Comment:
Started synchronizing cloud security groups with controller
```suggestion
	log.V(1).Info("Started synchronizing security groups in controller with cloud")
```